spec/unit/govuk_index/presenters/common_fields_presenter_spec.rb

```diff
@@ -124,6 +124,148 @@
     expect(presenter.popularity_b).to be_nil
   end
 
+  it "assigns political status when document is political" do
+    payload = generate_random_example(
+      schema: "news_article",
+      payload: {
+        document_type: "news_story",
+      },
+      details: {
+        political: true,
+      },
+    )
+
+    presenter = common_fields_presenter(payload)
+
+    expect(presenter.is_political).to eq(true)
```

Comment:
Minor but you switch from `eq(true)` to `be true` later in this test: https://github.com/alphagov/search-api/pull/1850/files#diff-3d91464b869a9b45ca7c4ac752b0e927R208

Reply:
Ah, the problem of writing code over several days!